### ISSUE TITLE
Missing (peer) dependency: chalk

### ISSUE THREAD

**Issue by michaelvogtmobi** [mockoon/mockoon#1276]
**Describe the bug**

I use mockoon in my project and tried to start it - this failed as you can see here:

```
# npx mockoon-cli start
    Error: require() of ES Module /repo/node_modules/chalk/source/index.js from /repo/node_modules/@mockoon/cli/dist/commands/start.js not supported.
    Instead change the require of index.js in /repo/node_modules/@mockoon/cli/dist/commands/start.js to a dynamic import() which is available in all CommonJS modules.
    Code: ERR_REQUIRE_ESM
```

**Steps to Reproduce**

- if the project that uses mockoon also use chalk v5, mockoon cannot be used anymore
- to fix this we should define chalk@4 as explicit dependency in the package.json or get rid of chalk(`packages/cli/src/commands/start.ts`)
- i think get rid of it might be the leaner and simpler solution - but open to suggestions
 
**Mockoon version:**

tested mockoon 5.1.0 and 6.1.0 - both failed.

**OS / OS version:**

Linux & OSX.

**Comment by 255kb** [comment]
Didn't expect this to happen. Thank you for reporting. 
You are right, getting rid of chalk is probably better. I will replace it by the color codes instead.

**Comment by 255kb** [comment]
📦 This is now fixed in [v6.2.0](https://mockoon.com/releases/6.2.0/).
